spec/system/task_spec.rb

```diff
@@ -1,39 +1,47 @@
 require 'rails_helper'
 RSpec.describe 'タスク管理機能', type: :system do
+  # before do
+  #   # あらかじめタスク一覧のテストで使用するためのタスクを二つ作成する
+  #   FactoryBot.create(:task)
+  #   FactoryBot.create(:second_task)
+  #   FactoryBot.create(:second_task, name: '付け加えた名前３', description: '付け加えたコンテント')
+  # end
+  before do
+    # あらかじめタスク一覧のテストで使用するためのタスクを二つ作成する
+    @task1 = FactoryBot.create(:task)
+    @task2 = FactoryBot.create(:second_task)
+    @task3 = FactoryBot.create(:second_task, name: '付け加えた名前３', description: '付け加えたコンテント')
+    # 「タスク一覧画面」や「タスク詳細画面」などそれぞれのテストケースで、before内のコードが実行される
+    # 各テストで使用するタスクを1件作成する
+    # 作成したタスクオブジェクトを各テストケースで呼び出せるようにインスタンス変数に代入
+    @task4 = FactoryBot.create(:task, name: 'task')
+  end
   describe 'タスク一覧画面' do
     context 'タスクを作成した場合' do
       it '作成済みのタスクが表示される' do
-        # テストで使用するためのタスクを作成
-        task = FactoryBot.create(:task, name: 'task')
-        # タスク一覧ページに遷移
         visit tasks_path
-        # visitした（遷移した）page（タスク一覧ページ）に「task」という文字列が
-        # have_contentされているか。（含まれているか。）ということをexpectする（確認・期待する）
         expect(page).to have_content 'task'
-        # expectの結果が true ならテスト成功、false なら失敗として結果が出力される
+      end
+    end
+    context '複数のタスクを作成した場合' do
+      it 'タスクが作成日時の降順に並んでいる' do
+        new_task = FactoryBot.create(:task, name: 'new_task')
```

Comment:
`config.include FactoryBot::Syntax::Methods`を定義していれば下記のように省略できます。
```suggestion
        new_task = create(:task, name: 'new_task')
```

Reply:
勉強になりました。ありがとうございます！

Reply:
コミットID：af241b0で修正しました。よろしくお願い致します。